### PR TITLE
CATROID-1416 "Find/search" in Scripts should ignore trailing spaces

### PR DESCRIPTION
**Pull request by devsebastian**
1. Ticket 1: https://jira.catrob.at/browse/CATROID-1416 -> [cc19d76](https://github.com/Catrobat/Catroid/pull/4535/commits/cc19d76a9859fa7028d6d9fb35b17587008e7f8a)
2. Ticket 2: https://jira.catrob.at/browse/CATROID-1415 -> [70f39df](https://github.com/Catrobat/Catroid/pull/4535/commits/70f39df5f1d30f8d691264f9c0ffdc23fb540710)

### Also:
1. added support to start search on pressing the enter key on the soft keyboard
2. Changed enter key icon to search instead of "next".
-> [22308cb](https://github.com/Catrobat/Catroid/pull/4535/commits/22308cb51f37d9ef0f3e2484071d0c5986212fad)

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ]  After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by Walcho1125 — CHANGES_REQUESTED** [review]
Looks good to me, however please make sure to update all File headers to "2010-2022"  and that there is only one commit on the PR!

File: catroid/src/androidTest/java/org/catrobat/catroid/test/stage/SearchParameterTest.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * Catroid: An on-device visual programming system for Android devices
- * Copyright (C) 2010-2021 The Catrobat Team
+ * Copyright (C) 2010-2022sc The Catrobat Team
```
Comment (Walcho1125) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/stage/SearchParameterTest.java:3]: please remove the accidently added "sc"

**Review by Walcho1125 — CHANGES_REQUESTED** [review]

File: catroid/src/androidTest/java/org/catrobat/catroid/test/stage/SearchParameterTest.java (hunk 1 above)
Comment (devsebastian) [reply catroid/src/androidTest/java/org/catrobat/catroid/test/stage/SearchParameterTest.java]: Oops sorry for the mixup. fixed it now

**Review by Walcho1125 — APPROVED** [review]
Looks good!

**Comment by wslany** [comment]
Thanks!!

**Pull request merged**